mathesar_ui/src/component-library/select/Select.svelte

```diff
@@ -190,7 +181,11 @@
   on:open={setSelectedItem}
 >
   <svelte:fragment slot="trigger">
-    {value?.[labelKey]}
+    {#if typeof value !== 'undefined'}
+      {getLabel(value, labelKey)}
+    {:else}
+      No option selected
```

Comment:
Displaying "No option selected" here is a little weird to me. But I don't think it's worth addressing now. We can fix this if/when it becomes a problem. I'd prefer to show it as blank.